lib/vsc/install/shared_setup.py

```diff
@@ -1158,11 +1158,6 @@ def cleanup(self, prefix=''):
                 except OSError:
                     log.error("cleanup failed for %s" % d)
 
-        for fn in ('setup.cfg',):
-            ffn = prefix + fn
-            if os.path.isfile(ffn):
-                os.remove(ffn)
```

Comment:
are you sure it will create one if one already exists without deleting it first?